### PR TITLE
New Extension: esa search

### PR DESCRIPTION
**Pull request by kawamataryo**
## Description

Add a new extensions esa search.
This extension allows you to search for your [esa](https://esa.io/) posts.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/esa-search/package.json [hunk 1]
```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "esa-search",
+  "title": "esa search",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:4]:
```suggestion
  "title": "esa Search",
```

File: extensions/esa-search/package.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  "author": "kawamataryo",
+  "categories": [
+    "Productivity",
+    "Documentation"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "searchPost",
+      "title": "Search esa Post",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:16]:
```suggestion
      "title": "Search Post",
      "subtitle": "esa",
```

File: extensions/esa-search/package.json [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+          "name": "keyword",
+          "placeholder": "keyword",
+          "type": "text",
+          "required": true
+        }
+      ]
+    },
+    {
+      "name": "createPost",
+      "title": "Create esa Post",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:30]:
```suggestion
      "title": "Create Post",
      "subtitle": "esa",
```

File: extensions/esa-search/package.json [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      "description": "Create post for esa.",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "title": "default show detail",
+      "name": "isShowDetail",
+      "description": "Show detail screen in initial view",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:39]:
```suggestion
      "description": "	Show Detail Screen in Initial View",
```

File: extensions/esa-search/package.json [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+      "type": "dropdown",
+      "default": "false",
+      "data": [
+        {
+          "value": "true",
+          "title": "true"
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:45]:
```suggestion
          "title": "Yes"
```

File: extensions/esa-search/package.json [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+        },
+        {
+          "value": "false",
+          "title": "false"
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:49]:
```suggestion
          "title": "No"
```

File: extensions/esa-search/package.json [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+        }
+      ],
+      "required": false
+    },
+    {
+      "title": "team name",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:55]:
```suggestion
      "title": "Team Name",
```

File: extensions/esa-search/package.json [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+      "name": "primaryTeamName",
+      "description": "Your esa team name",
+      "type": "textfield",
+      "placeholder": "esa",
+      "required": true
+    },
+    {
+      "title": "token",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:63]:
```suggestion
      "title": "Token",
```

File: extensions/esa-search/package.json [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+      "name": "primaryToken",
+      "description": "Your esa access token",
+      "type": "textfield",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:66]: Might be a password field?

File: extensions/esa-search/package.json [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+      "placeholder": "xxx",
+      "required": true
+    },
+    {
+      "title": "secondary team name",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:71]:
```suggestion
      "title": "Secondary Team Name",
```

File: extensions/esa-search/package.json [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+      "name": "secondaryTeamName",
+      "description": "Your esa team name",
+      "type": "textfield",
+      "placeholder": "esa",
+      "required": false
+    },
+    {
+      "title": "secondary token",
```
Comment (pernielsentikaer) [inline extensions/esa-search/package.json:79]:
```suggestion
      "title": "Secondary Token",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I added some feedback for you to address 🔥 

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by kawamataryo** [comment]
@pernielsentikaer 

Thank you for your review 👍
I fixed.
https://github.com/raycast/extensions/pull/2567/commits/376b8b16bb4814f6d15739361db9b5ebc5f512dd.

Please review again.

File: extensions/esa-search/package.json (hunk 4 above)
Comment (kawamataryo) [reply extensions/esa-search/package.json:41]: As far as the other commands are concerned, the `description` field is lower case, should this be changed?

Comment (pernielsentikaer) [reply extensions/esa-search/package.json:41]: My bad, wrong line 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**